ibm_mq/tests/conftest.py

```diff
@@ -0,0 +1,61 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+import time
+import subprocess
+import pytest
+
+from datadog_checks.dev import get_docker_hostname
+from datadog_checks.dev import docker_run
+from datadog_checks.ibm_mq import IbmMqCheck
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+COMPOSE_DIR = os.path.join(HERE, 'compose')
+
+
+@pytest.fixture
+def check():
+    return IbmMqCheck('ibm_mq', {}, {})
+
+
+@pytest.fixture
+def instance():
+    return {
+        'channel': 'DEV.ADMIN.SVRCONN',
+        'queue_manager': 'datadog',
+        'host': get_docker_hostname(),
+        'port': '11414',
+        'username': 'admin',
+        'password': 'passw0rd',
+        'queues': [
+            'DEV.QUEUE.1'
+        ]
+    }
+
+
+@pytest.fixture(scope='session')
+def spin_up_ibmmq():
+    mq_version = os.environ.get('IBM_MQ_VERSION', '9')
+    compose_file_name = 'docker-compose-v{}.yml'.format(mq_version)
+    env = {
+        'COMPOSE_DIR': COMPOSE_DIR
+    }
+
+    if mq_version == '9':
```

Comment:
Need a default value for `log_pattern` otherwise it may never get defined

Reply:
if we add other versions we'll add it then. it should fail if it's not one of the approved versions because there's no default